### PR TITLE
Feature implement event report tables

### LINKED ISSUE #15: Implement Event Report Tables

### PR DESCRIPTION
**Pull request by manuelplazaspalacio**
Close #15

### CODE REVIEW

File: api/src/main/java/org/hisp/dhis/android/dashboard/api/network/DhisApi.java [hunk 1]
```diff
@@ -141,6 +142,17 @@ Response deleteDashboardItemContent(@Path("dashboardUid") String dashboardUid,
     @GET("/reportTables/{id}/data.html")
     Response getReportTableData(@Path("id") String id);
 
+    @Headers("Accept: application/json")
+    @GET("/eventReports/{id}")
+    EventReport getEventReport(@Path("id") String id);
+
+    @Headers("Accept: application/text")
+    @GET("/25/analytics/events/query/{program}"
```
Comment (xurxodev) [inline api/src/main/java/org/hisp/dhis/android/dashboard/api/network/DhisApi.java]: can we create request without version segment as another request?

**Review by xurxodev — CHANGES_REQUESTED** [review]
@manuelplazaspalacio It is not working, when I navigate to event report detail screen appears a ProgressBar but when this ProgressBar is finished, the screen is empty.
Can we create request without version 25 segment as another request?

**Comment by xurxodev** [comment]
@manuelplazaspalacio the items of type event report should have an event report icon

**Comment by manuelplazaspalacio** [comment]
Changes requests done @xurxodev

**Review by xurxodev — APPROVED** [review]
thanks @manuelplazaspalacio

**Comment by xurxodev** [comment]
@manuelplazaspalacio The image looked wrong, I have solved it by modifying the scale type and adding other larger ones that I have found in dhis server

**Pull request merged**